### ISSUE TITLE
Constant crash of the world server

### ISSUE THREAD

**Issue by MRafStudio** [liyunfan1223/mod-playerbots#51]
My feeling is that it's a memory leak. At least it complains about stack overflow. Each crash occurs after the server eats up more than 6 gigabytes of memory. All bot settings are by default. The server is compiled in debug mode. I am attaching the files.
[a5e27362d8b5_worldserver.exe_[24-10_15-11-25].txt](https://github.com/liyunfan1223/mod-playerbots/files/13113153/a5e27362d8b5_worldserver.exe_.24-10_15-11-25.txt)
[a5e27362d8b5_worldserver.exe_[24-10_15-11-25].dmp](https://github.com/liyunfan1223/mod-playerbots/files/13113154/a5e27362d8b5_worldserver.exe_.24-10_15-11-25.dmp)

**Comment by MRafStudio** [comment]
I can add that I use the following modules:
1v1arena
mod_ahbot
mod_LuaEngine
mod_pvptitles
npc_beastmaster

**Comment by MRafStudio** [comment]
I compile the kernel and bots from the following repositories:
AzerothCore:
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = https://github.com/liyunfan1223/azerothcore-wotlk.git
	fetch = +refs/heads/Playerbot:refs/remotes/origin/Playerbot
[branch "Playerbot"]
	remote = origin
	merge = refs/heads/Playerbot
[gui]
	wmstate = normal
	geometry = 893x435+156+156 175 196

Playerbots:
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = https://github.com/liyunfan1223/mod-playerbots.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master

**Comment by MRafStudio** [comment]
I still don’t understand how I solved the problem, but in a nutshell:
1. Reassigned the Build folder to a new folder and recompiled the project.
2. Instead of manually messing with the sql module files, I copied them to the corresponding data/sql/custom folders
3. After thinking, I increased the parameter PlayerLimit = 5000 (worldserver.conf)
4. I launched both the server and MySQL on the SSD (the speed of everything increased quite noticeably)
***
As a result, everything works stably, the memory is loaded up to 4 GB and reset to 2.
***
There was already an idea to reduce the value of the AiPlayerbot.IterationsPerTick parameter in playerbot.conf to 5 and increase AiPlayerbot.ReturnDelay to 3000, but since everything worked, I left it as is. Such miracles.
